app/src/main/java/org/mifos/mobile/viewModels/RecentTransactionViewModel.kt

```diff
@@ -12,42 +12,47 @@ import org.mifos.mobile.utils.RecentTransactionUiState
 import javax.inject.Inject
 
 @HiltViewModel
-class RecentTransactionViewModel @Inject constructor(private val recentTransactionRepositoryImp: RecentTransactionRepository) :
+class RecentTransactionViewModel @Inject constructor(private val recentTransactionRepository: RecentTransactionRepository) :
```

Comment:
dont change the name of the variable , it was set to `recentTransactionRepositoryImp` deliberately